### PR TITLE
Create FRA example notebook

### PR DESCRIPTION
**Pull request by andrewjlp**
# Pull Request Checklist

- [ ] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [ ] Tests pass
- [ ] Raised the PR against the `develop` branch

# Description of the PR
https://finbourne.atlassian.net/browse/SE-2374

New Forward Rate Agreement instrument notebook in: instruments

- Booking transaction
- Creating market data quotes for discount and forward rates
- Creating recipe
- Valuations at multiple points in lifecycle
- Fix and book cashflows

### CODE REVIEW

File: examples/use-cases/instruments/Forward Rate Agreement.ipynb [hunk 1]
```diff
@@ -0,0 +1,1613 @@
+{
```
Comment (cscialino) [inline examples/use-cases/instruments/Forward Rate Agreement.ipynb:1]: <p>Do we need the FX vol stuff here? Perhaps we can just not display it if it's the same table from other examples.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (cscialino) [inline examples/use-cases/instruments/Forward Rate Agreement.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #20.            valuation_schedule=lm.ValuationSchedule(effective_at=date),
</pre><p>This is where you can add the time interval.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (cscialino) [inline examples/use-cases/instruments/Forward Rate Agreement.ipynb:1]: <p>Valuation/PV/Amount vs. Market Value, both needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (cscialino) [inline examples/use-cases/instruments/Forward Rate Agreement.ipynb:1]: <p>You can actually do this more natively with the APIs, you can pass in a <em>ValuationSchedule</em> object with a start and end date. (Also tenor, which is the frequency in this case.)</p><p> </p><p><a href="https://github.com/finbourne/lusid-sdk-python-preview/blob/master/sdk/lusid/models/valuation_schedule.py#L25" target="_blank">lusid-sdk-python-preview/valuation_schedule.py at master · finbourne/lusid-sdk-python-preview (github.com)</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (cscialino) [inline examples/use-cases/instruments/Forward Rate Agreement.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #20.    fra_version = fra_version +1
</pre><p>*fra_version + 1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (cscialino) [inline examples/use-cases/instruments/Forward Rate Agreement.ipynb:1]: <p>Could we also run an AtoB report here? See the IRS notebook for examples.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (andrewjlp) [reply examples/use-cases/instruments/Forward Rate Agreement.ipynb:1]: Agree, I will hide it in the display

Comment (andrewjlp) [reply examples/use-cases/instruments/Forward Rate Agreement.ipynb:1]: Valuation/PV/Amount looks to be in instrument currency, and marketValue in portfolio currency, so I prefer to retain both for sanity. MarketValue is the key measure.

Comment (andrewjlp) [reply examples/use-cases/instruments/Forward Rate Agreement.ipynb:1]: ah i wondered how you did ++. I'll change

Comment (andrewjlp) [reply examples/use-cases/instruments/Forward Rate Agreement.ipynb:1]: I have added this

Comment (andrewjlp) [reply examples/use-cases/instruments/Forward Rate Agreement.ipynb:1]: That's done and added.

Comment (cscialino) [reply examples/use-cases/instruments/Forward Rate Agreement.ipynb:1]: <p>Then let's rename the columns accordingly. In fact, the <code>get_valuation</code> function has a renaming that is currently not being applied on the df columns. Can we amend this please? </p>

**Review by cscialino — APPROVED** [review]

**Pull request merged**